float64 ang_x
float64 ang_y
float64 ang_z
float64 linear_x
float64 linear_y
float64 linear_z